packages/sanity/src/core/studio/Studio.tsx

```diff
@@ -0,0 +1,30 @@
+import {ThemeColorSchemeKey} from '@sanity/ui'
```

Comment:
Extracted from somewhere, or moved?

Reply:
I refactored the studio props slightly, to avoid having to use `Omit`